<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DuME</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="progressWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Output size: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="outputSizeLabel">
           <property name="text">
            <string>0 MB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Output bitrate: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="outputBitrateLabel">
           <property name="text">
            <string>0 Mbps</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="currentEncodingNameLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Exepected size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="expectedSizeLabel">
           <property name="text">
            <string>0 MB</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Time elapsed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLabel">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="speedTitleLabel">
           <property name="text">
            <string>Speed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="speedLabel">
           <property name="text">
            <string>0x</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Time Remaining:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeRemainingLabel">
           <property name="text">
            <string>00:00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainWidget">
       <layout class="QVBoxLayout" name="mainLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QVBoxLayout" name="duqf_settingsLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Program and FFMpeg Settings</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/encode</normaloff>:/icons/encode</iconset>
   </property>
   <property name="text">
    <string>Go!</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ready.</string>
   </property>
  </action>
  <action name="actionBug_report">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/bugreport</normaloff>:/icons/bugreport</iconset>
   </property>
   <property name="text">
    <string>Bug report</string>
   </property>
   <property name="toolTip">
    <string>File a bug report to help the improvement of the application</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/chat</normaloff>:/icons/chat</iconset>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
   <property name="toolTip">
    <string>Let's have a chat on our Discord server</string>
   </property>
  </action>
  <action name="actionForum">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/forum</normaloff>:/icons/forum</iconset>
   </property>
   <property name="text">
    <string>Forum</string>
   </property>
   <property name="toolTip">
    <string>Come and have a chat on the forum</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/doc</normaloff>:/icons/doc</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Get help using this application</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionGoQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/encode2</normaloff>:/icons/encode2</iconset>
   </property>
   <property name="text">
    <string>Go! (quit when finished)</string>
   </property>
   <property name="toolTip">
    <string>Automatically quits DuME after the transcoding process</string>
   </property>
  </action>
  <action name="actionDevBuild">
   <property name="text">
    <string>Development Build</string>
   </property>
  </action>
  <action name="actionAbout_FFmpeg">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/ffmpegl</normaloff>:/icons/ffmpegl</iconset>
   </property>
   <property name="text">
    <string>About FFmpeg</string>
   </property>
  </action>
  <action name="actionShowQueue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/show-rqueue</normaloff>:/icons/show-rqueue</iconset>
   </property>
   <property name="text">
    <string>Queue</string>
   </property>
   <property name="toolTip">
    <string>Show/hide queue</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionLaunchJob">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/encode-job</normaloff>:/icons/encode-job</iconset>
   </property>
   <property name="text">
    <string>Launch job</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/tools</normaloff>:/icons/tools</iconset>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
